app/main.py

```diff
@@ -1,3 +1,57 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from decimal import Decimal
+
+from app.customer import Customer, Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        json_data = json.load(file)
+    Car.change_fuel_price(json_data["FUEL_PRICE"])
+    customers = [
```

Comment:
you can optimize this by having customers created and do all the shop trip related things in one loop. So you don't have to iterate over same data twice